sdk/core/azure-core/azure/core/configuration.py

```diff
@@ -89,7 +89,7 @@ def __init__(self, transport=None, **kwargs):
         self.transport = transport
 
         # Polling interval if no retry-after in polling calls results
-        self.polling_interval = 30
+        self.polling_interval = kwargs.get("lro_polling_interval", 30)
```

Comment:
I would suggest using `polling_interval` as the name. Sorry if I have caused any confusion around the naming :).